config/production/params.toml

```diff
@@ -38,15 +38,20 @@ no = 'Sorry to hear that. Please <a href="https://github.com/knative/docs/issues
 # Set to "docs" path
 
 [[versions]]
-  version = "v0.14"
-  ghbranchname = "release-0.14"
+  version = "v0.15"
+  ghbranchname = "release-0.15"
   url = "/docs/"
   dirpath = "docs"
 
 # Archived versions
 ###################
 # Prefix all past versions with release # "v#.#-docs"
 
+[[versions]]
+  version = "v0.14"
+  ghbranchname = "release-0.14"
+  url = "/v0.13-docs/"
```

Comment:
Should this and the line below be v0.14-docs?

Reply:
This is the archive section, so it should start with v14.  I've updated the url property so that it says 14 as well.  The 15 stuff is above, as it is the new current version